applications_tests/cfd_dem_coupling_3d/dynamic_contact_search_files/dynamic_contact_search_generator.prm

```diff
@@ -3,93 +3,100 @@
 # --------------------------------------------------
 # Simulation and IO Control
 #---------------------------------------------------
+
 subsection simulation control
-  set time step                 			 = 0.0001
-  set time end       					 = 0.0001
-  set log frequency				         = 1000
-  set output frequency            			 = 1
-  set output path                                      = ./output_dem/
+  set time step        = 0.0001
+  set time end         = 0.0001
+  set log frequency    = 1000
+  set output frequency = 1
+  set output path      = ./output_dem/
 end
 
 #---------------------------------------------------
 # Timer
 #---------------------------------------------------
+
 subsection timer
-    set type    					 = none
+  set type = none
```

Comment:
Is it necessary to keep it since its the default parameter for the timer subsection ?

Reply:
This file was modified with the prm indent.